packages/main/src/TableVirtualizer.ts

```diff
@@ -149,7 +149,7 @@ class TableVirtualizer extends UI5Element implements ITableFeature {
 	}
 
 	onExitDOM(): void {
-		this._scrollContainer.removeEventListener("scroll", this._onScrollBound);
+		this._scrollContainer?.removeEventListener("scroll", this._onScrollBound);
```

Comment:
same here since this component can only be added to the table, if somebody is removing this node from the dom tree then it must be already in the table which must have the scroll container.